DataDiscrepancyUtil/dataDiffCheck.py

```diff
@@ -1,61 +1,60 @@
+"""
+    Usage:
+    dataDiffCheck.py <old_data_path> <new_data_path>
+
+    Arguments:
+        old_data_path: File path of old building json
+        new_data_path: File path of new building json
+"""
+# old.json being the data from https://api.oregonstate.edu/v1/locations?page[size]=10000&type=building
+# new.json being the data from https://localhost:8082/api/v0/locations?page[size]=10000&type=building - locations-frontend-api commit b1bec1e013cf29f16a0a01b9dd7c3777e3b4e192
+
+from docopt import docopt
+print(docopt(__doc__, version='1.0.0rc2'))
+
+args = docopt(__doc__, version='1.0.0rc2')
+
```

Comment:
According to PEP8 standard, module level import should be at top of the file. See error [E402](http://pep8.readthedocs.io/en/release-1.7.x/intro.html#error-codes).